applications/rag/variables.tf

```diff
@@ -31,6 +31,11 @@ variable "cluster_location" {
   type = string
 }
 
+variable "kubernetes_version" {
+  type    = string
+  default = "1.28"
```

Comment:
Does this version still use DPv2? Assuming this was the networking issue we saw, I'm not sure that just downgrading to v1.28 will help

Reply:
It seems there are multiple features at play. Looks like this satisfies our API requirements and will likely be more stable.